Makefile

```diff
@@ -39,6 +39,8 @@ gt = $(and $(call ge,$(1),$(2)),$(call ne,$(1),$(2)))
 KERNEL_FLAVOR := $(if $(call gt,$(SERIES_RELEASE),18.04),raspi,raspi2)
 FIRMWARE_FLAVOR := $(if $(call ge,$(SERIES_RELEASE),22.04),raspi,raspi2)
 
+KERNEL_VERSION := $(shell linux-version list | grep $(KERNEL_FLAVOR) | linux-version sort | tail -1)
```

Comment:
isn't this going to be the kernel version installed in the host?

Reply:
Yes, pushed an update to fix.